Mimir/GraphQL/Types/AgentType.cs

```diff
@@ -9,9 +13,21 @@ public class AgentType : ObjectType<AgentObject>
 {
     protected override void Configure(IObjectTypeDescriptor<AgentObject> descriptor)
     {
+        descriptor
+            .Field("version")
+            .Type<IntType>()
+            .Resolve(context => GetAgent(context)?.Version ?? null);
         descriptor
             .Field(f => f.Address)
             .Type<NonNullType<AddressType>>();
+        descriptor
+            .Field("avatarAddresses")
+            .Type<NonNullType<ListType<NonNullType<AddressType>>>>()
+            .Resolve(context => GetAgent(context)?.AvatarAddresses.Values.ToArray() ?? []);
+        descriptor
+            .Field("monsterCollectionRound")
+            .Type<IntType>()
+            .Resolve(context => GetAgent(context)?.MonsterCollectionRound ?? null);
```

Comment:
This field may be deprecated since `MonsterCollection` is deprecated. It uses `Stake` now.

```suggestion
```

Reply:
Yeah I know, it's just a value that's being stored so anyone can query it. I'll add a description so they know it's expired.

Reply:
Looking at the "stake3" action now, I see that there is some handling logic for the unmigrated staking state. It seems like a good idea to leave it undeprecated for now, rather than considering more background.
https://github.com/planetarium/lib9c/blob/v200180/Lib9c/Action/Stake.cs#L56-L62